src/public_key.rs

```diff
@@ -3,8 +3,7 @@
 //! since a client will need to be able to parse and use a public key from any
 //! keypair.
 use crate::*;
-use std::convert::TryFrom;
-
+use std::{convert::TryFrom, hash::Hash};
```

Comment:
No need to move this down a line, now is there

Reply:
Gotta maximize lines removed in PRs for street cred